docs/administration/Configuration.md

```diff
@@ -42,462 +42,2042 @@ ADMIN SET FRONTEND CONFIG ("key" = "value");
+
+- **Default:** 10 seconds
+- **Description:** The time interval at which to report the state of a task. A task can be creating a table, dropping a table, loading data, or changing a table schema.
+
+#### report_disk_state_interval_seconds
+
+- **Default:** 60 seconds
+- **Description:** The time interval at which to report the storage volume state, which includes the size of data within the volume.
+
+#### report_tablet_interval_seconds
+
+- **Default:** 60 seconds
+- **Description:** The time interval at which to report the most updated version of all tablets.
+
+#### report_workgroup_interval_seconds
+
+- **Default:** 5 seconds
+- **Description:** The time interval at which to report the most updated version of all workgroups.
+
+#### max_download_speed_kbps
+
+- **Default:** 50,000 KB/s
+- **Description:** The maximum download speed of each HTTP request. This value affects the performance of data replica synchronization across BE nodes.
+
+#### download_low_speed_limit_kbps
+
+- **Default:** 50 KB/s
+- **Description:** The download speed lower limit of each HTTP request. An HTTP request aborts when it constantly runs with a lower speed than this value within the time span specified in the configuration item `download_low_speed_time`.
+
+#### download_low_speed_time
+
+- **Default:** 300 seconds
+- **Description:** The maximum time that an HTTP request can run with a download speed lower than the limit. An HTTP request aborts when it constantly runs with a lower speed than the value of `download_low_speed_limit_kbps` within the time span specified in this configuration item.
+
+#### status_report_interval
+
+- **Default:** 5 seconds
+- **Description:** The time interval at which a query reports its profile, which can be used for query statistics collection by FE.
+
+#### scanner_thread_pool_thread_num
+
+- **Default:** 48 (Number of Threads)
+- **Description:** The number of threads which the storage engine used for concurrent storage volume scanning. All threads are managed in the thread pool.
+
+#### thrift_client_retry_interval_ms
+
+- **Default:** 100 ms
+- **Description:** The time interval at which a thrift client retries.
+
+#### scanner_thread_pool_queue_size
+
+- **Default:** 102,400 (Number of Tasks)
+- **Description:** The number of scan tasks supported by the storage engine.
+
+#### scanner_row_num
+
+- **Default:** 16,384 (Maximum Rows per Scan Thread)
+- **Description:** The maximum row count returned by each scan thread in a scan.
+
+#### max_scan_key_num
+
+- **Default:** 1,024 (Maximum Number of Scan Keys per Query)
+- **Description:** The maximum number of scan keys segmented by each query.
+
+#### max_pushdown_conditions_per_column
+
+- **Default:** 1,024 (Maximum Number of Pushdown Conditions per Column)
+- **Description:** The maximum number of conditions that allow pushdown in each column. If the number of conditions exceeds this limit, the predicates are not pushed down to the storage layer.
```

Comment:
![image](https://github.com/StarRocks/starrocks/assets/57167462/5204a217-8aa7-4d73-9d8b-60d6860165ad)
All these should be deleted, I think.